tenzir.yaml.example

```diff
@@ -270,6 +270,11 @@ tenzir:
       # - Set the option to a valid duration to enable with a custom delay.
       restart-on-error: 1 minute
 
+
+  # The path from which the node attempts to read statically configured packages.
+  # This setting is ignored unless the pipeline manager plugin is enabled.
+  # packages-dir: <sysconfir>/packages
+
```

Comment:
Instead of making this configurable, can you make this a fixed `<configdir>/packages`, and please also support the other config directories beside the sysconfdir?

Reply:
Is there a use case for this? If not I'd prefer not to, since it's a lot of additional work since we'd have to collect the set of all packages from all  configdirs, handle conflicts, etc.

Setting the package dir directly has also proven useful so far, e.g. for installing some commit of the whole library for testing purposes, or for easily overriding it in tests.

Reply:
> Setting the package dir directly has also proven useful so far, e.g. for installing some commit of the whole library for testing purposes, or for easily overriding it in tests.

I'm not against having it configurable, but even then it should probably work similarly to `--config`, `--plugin-dirs` and `--schema-dirs`, where it simply adds a directory with the highest precedence instead of disabling the default directories, and disabling the default directories requires setting `--bare-mode`.

Reply:
How about just *not* setting a default value then? Then we can avoid the whole issue and for the case where we actually want to use this for deploying packages via IaC we're controlling the node config anyways and can just set the directory.

Reply:
But why have this behave different from the other config subdirectories?